assets/skins/DefenceDefault.skin

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
this should have different name

Reply:
if we believe this is a good default for ... shop screens ... we should just add the skin to Economy.